src/middlewares/Fetch.spec.ts

```diff
@@ -2,20 +2,27 @@ import {IRequest, IRequestHeaders} from "../Client";
 import {Fetch} from "./Fetch";
 
 describe("Fetch", () => {
+    describe("is defined and is available", () => {
+        it("should be defined", () => {
+            const fetch = new Fetch();
+            expect(fetch).toBeDefined();
+        });
```

Comment:
Since we're wrapping/testing the use of the empty constructor within the construct()-block, it should be reasonably be enough here to just do `expect(Fetch).toBeDefined();` as it will ensure there was something imported under the given name.

The new Fetch() might fail for various reasons resulting in a misleading test result (aka. "WTF?! Why is it failing?! I did import it!!!!")

So just test for the imported object to exist, Don't actually invoke anything on it at this point :) 

PS: This should be considered standard in the same way as the comment above. We can discuss this tuesday morning if you like